ivadomed/scripts/curate_data_demo.py

```diff
@@ -0,0 +1,224 @@
+"""
+The data are structure in a folder such as:
+dataset/
+├── 20220309_demo_mouse_07_0001
+│         ├── image.png
+│         ├── mask.png
+│         ├── mask_seg-axon-manual.png
+│         ├── mask_seg-myelin-manual.png
+│         └── pixel_size_in_micrometer.txt
+├── 20220309_demo_mouse_07_0002
+│         ├── image.png
+│         ├── mask.png
+│         ├── mask_seg-axon-manual.png
+│         ├── mask_seg-myelin-manual.png
+│         └── pixel_size_in_micrometer.txt
+└── 20220309_demo_mouse_07_0003
+          ├── image.png
+          ├── mask.png
+          ├── mask_seg-axon-manual.png
+          ├── mask_seg-myelin-manual.png
+          └── pixel_size_in_micrometer.txt
+
+"""
+
+# Define various dependencies
+
+import glob
+import os
+import shutil
+import json
+import argparse
+import subprocess
+import csv
+from textwrap import dedent
+
+## Dictionary for images
+images = {
+    "image.png": "_TEM.png"
+}
+
+## Dictionary for derivatives
+der = {
+    "mask.png": "_TEM_seg-axonmyelin-manual.png",
+    "mask_seg-axon-manual.png": "_TEM_seg-axon-manual.png",
+    "mask_seg-myelin-manual.png": "_TEM_seg-myelin-manual.png"
+}
+
+# Define function to get the input and output path for data
+def get_parameters():
+    parser = argparse.ArgumentParser(description='This script is curating dataset data_axondeepseg_tem to BIDS')
+    # Define input path
+    parser.add_argument("-d", "--data",
+                        help="Path to folder containing the dataset to be curated",
+                        required=True)
+    # Define output path
+    parser.add_argument("-o", "--outputdata",
+                        help="Path to output folder",
+                        required=True,
+                        )
+    arguments = parser.parse_args()
+    return arguments
+
+# Define function to create json sidecar
+def create_json_sidecar(output_data, sub_id):
+
+    # Path for each subject  destination bids folder
+    path_folder_sub_id_bids = os.path.join(output_data, sub_id, 'microscopy')
```

Comment:
```suggestion
    path_folder_sub_id_bids = os.path.join(output_data, sub_id, 'micr')
```

`microscopy` is outdated, the data type is `micr` in the released spec.